ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Ledger/Query.hs

```diff
@@ -135,6 +139,22 @@ instance All SingleEraBlock xs => BlockSupportsLedgerQuery (HardForkBlock xs) wh
       lcfg = configLedger cfg
       ei   = State.epochInfoLedger lcfg hardForkState
 
+  blockQueryIsSupportedOnVersion q (HardForkNodeToClientDisabled x) = case q of
+    QueryIfCurrent (QZ q') -> blockQueryIsSupportedOnVersion q' x
+    QueryIfCurrent{}       -> False
+    QueryAnytime{}         -> False
+    QueryHardFork {}       -> False
+  blockQueryIsSupportedOnVersion q (HardForkNodeToClientEnabled hfv npversions) = case q of
+    QueryIfCurrent qc -> hfv >= HardForkSpecificNodeToClientVersion3 &&
+      go qc npversions
+    QueryAnytime{} -> hfv >= HardForkSpecificNodeToClientVersion3
+    QueryHardFork{} -> hfv >= HardForkSpecificNodeToClientVersion3
```

Comment:
I think all of these were actually added before `HardForkSpecificNodeToClientVersion3`, so I think you can replace these `hfv >= HardForkSpecificNodeToClientVersion3` with `True`.